packages/zoe/src/contractSupport/priceAuthority.js

```diff
@@ -1,44 +1,46 @@
+// @ts-check
+
 import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import { assert, details as X } from '@agoric/assert';
 import { makePromiseKit } from '@agoric/promise-kit';
+import { amountMath } from '@agoric/ertp';
 
 import '../../exported';
 
 /**
  * @callback CompareAmount
- * @param {DeprecatedAmountMath} math
  * @param {Amount} amount
  * @param {Amount} amountLimit
  * @returns {boolean}
  */
 
 /** @type {CompareAmount} */
-const isLT = (math, amountOut, amountLimit) =>
-  !math.isGTE(amountOut, amountLimit);
+const isLT = (amountOut, amountLimit) =>
+  !amountMath.isGTE(amountOut, amountLimit);
 
 /** @type {CompareAmount} */
-const isLTE = (math, amount, amountLimit) => math.isGTE(amountLimit, amount);
+const isLTE = (amount, amountLimit) => amountMath.isGTE(amountLimit, amount);
 
 /** @type {CompareAmount} */
-const isGTE = (math, amount, amountLimit) => math.isGTE(amount, amountLimit);
+const isGTE = (amount, amountLimit) => amountMath.isGTE(amount, amountLimit);
 
 /** @type {CompareAmount} */
-const isGT = (math, amount, amountLimit) => !math.isGTE(amountLimit, amount);
+const isGT = (amount, amountLimit) => !amountMath.isGTE(amountLimit, amount);
 
 /**
  * @typedef {Object} OnewayPriceAuthorityOptions
  * @property {Issuer} quoteIssuer
- * @property {DeprecatedAmountMath} mathIn
- * @property {DeprecatedAmountMath} mathOut
  * @property {Notifier<PriceQuote>} notifier
  * @property {TimerService} timer
  * @property {PriceQuoteCreate} createQuote
+ * @property {Brand} actualBrandIn
+ * @property {Brand} actualBrandOut
  */
 
 /**
  * @callback Trigger
- * @param {Timestamp} timestamp
+ * @param {Function} createInstantQuote
```

Comment:
is `Trigger` different from `PriceQuoteTrigger`?

Reply:
Yeah, Trigger has the method `createInstanceQuote` but `PriceQuoteTrigger` has `createQuote`

Reply:
`Function`?
```suggestion
 * @param {PriceQuoteTrigger} createInstantQuote
```
I think. But almost certainly not `Function`.

Reply:
Ah, this is actually a function. I think it's a `PriceQuoteCreate`